misk-hotwire/src/main/kotlin/misk/turbo/BuildHtml.kt

```diff
@@ -0,0 +1,15 @@
+package misk.turbo
+
+import kotlinx.html.TagConsumer
+import kotlinx.html.stream.appendHTML
+
+fun buildHtml(renderer: TagConsumer<*>.() -> Unit) = StringBuilder().apply {
+  appendHTML().renderer()
+}.toString()
+
+// TODO replace with ResponseBody
```

Comment:
What is blocking this?

Reply:
I added a commit and will land any feedback in follow up PRs.